Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "odra-vm",
     "core",
     "odra-macros",
+    "odra-casper/casper-client",
```

Comment:
Why twice `odra-casper/casper-client`?
BTW, this name is a bit weird. Why don't we rename this crate to `odra-casper-rpc-client` in `odra-casper/rpc-client` directory?

Reply:
Ok